README.org

```diff
@@ -12,25 +23,36 @@ It allows you to specify a list of files you want deleted on hosts in your infra
 
 #+caption: Example Augments configuration
 #+begin_src json
-  {
-    "variables": {
-      "delete_files:delete_files.files": {
-        "value": [
-          {
-            "path": "/etc/hosts.equiv",
-            "why": "The file lists remote hosts and users that are trusted by the local system when using the rshd daemon and can allow unauthenticated access to the system. Reccomended by CIS  CCE-84145-2."
-          },
-         {
-            "path": "/home/.*/.rhosts",
-            "why": "The file lists remote hosts and users that are trusted by the local system when using the rshd daemon and can allow unauthenticated access to the system. Reccomended by CIS  CCE-84145-2."
-          }
-          {
-            "path": "/root/.rhosts",
-            "why": "The file lists remote hosts and users that are trusted by the local system when using the rshd daemon and can allow unauthenticated access to the system. Reccomended by CIS  CCE-84145-2."
-          }
-        ],
-        "comment": "Files and their required permissions"
-      }
+{
+  "variables": {
+    "delete_files:delete_files.files": {
+      "value": [
+        {
+          "path": "/tmp/virus",
+          "why": "This is just an example, but if there is a file called virus there, you'd want it deleted!"
+        },
+        {
+          "path": "/home/.*/.shosts",
+          "why": "These files list remote hosts and users for host based authentication. Deletion recommended by CIS CCE-84145-2."
+        },
+        {
+          "path": "/etc/cron.deny",
+          "why": "Deny should be default, use an explicit allow list in /etc/cron.allow instead."
+        }
+      ],
+      "comment": "Files and their required permissions"
     }
   }
+}
+#+end_src
+
+** Testing that it works
+
+After adding the file paths you want deleted (via input or JSON) you can touch the files and see that they get deleted:
+
+#+begin_src
```

Comment:
```suggestion
#+begin_example
```

This is more of an example block than a src block that might get tangled to another file or directly executed.